bazel/swift_header_collector.bzl

```diff
@@ -0,0 +1,23 @@
+"""
+Propagate the generated Swift header from a swift_library target
+
+This exists to work around https://github.com/bazelbuild/rules_swift/issues/291
+"""
+
+def _swift_header_collector(ctx):
+    dep = ctx.attr.deps[0]
+    return [
+        DefaultInfo(
+            files = dep[CcInfo].compilation_context.headers,
```

Comment:
AFAIK since this is a direct dep there cannot be more than 1 in this list, but I could filter by ones ending with `-Swift.h` if that wasn't the case.

Reply:
From what I saw in the past we only had one here IIRC, so this is prob fine